### PR TITLE
Update multiple packages

### LINKED ISSUE #699: Update github.com/justwatchcom/elasticsearch_exporter to v1.2.1
An update for `github.com/justwatchcom/elasticsearch_exporter` (version `v1.2.1`) is now available. Version `v1.1.0` is currently in use.

### LINKED ISSUE #730: Update github.com/wrouesnel/postgres_exporter to v0.10.0
An update for `github.com/wrouesnel/postgres_exporter` (version `v0.10.0`) is now available. Version `v0.8.0` is currently in use.

### LINKED ISSUE #1109: Update github.com/ncabatoff/process-exporter to v0.7.10
An update for `github.com/ncabatoff/process-exporter` (version `v0.7.10`) is now available. Version `v0.7.5` is currently in use.

### LINKED ISSUE #1380: Update github.com/prometheus/consul_exporter to v0.8.0
An update for `github.com/prometheus/consul_exporter` (version `v0.8.0`) is now available. Version `v0.7.2-0.20210127095228-584c6de19f23` is currently in use.

### LINKED ISSUE #1466: Update github.com/prometheus/mysqld_exporter to v0.14.0
An update for `github.com/prometheus/mysqld_exporter` (version `v0.14.0`) is now available. Version `v0.13.0` is currently in use.

### LINKED ISSUE #1813: Update github.com/prometheus/memcached_exporter to v0.10.0
An update for `github.com/prometheus/memcached_exporter` (version `v0.10.0`) is now available. Version `v0.9.0` is currently in use.

### LINKED ISSUE #1970: Update github.com/prometheus-community/elasticsearch_exporter to v1.5.0
An update for `github.com/prometheus-community/elasticsearch_exporter` (version `v1.5.0`) is now available. Version `v1.2.1` is currently in use.

### LINKED ISSUE #2148: Update github.com/prometheus/statsd_exporter to v0.22.8
An update for `github.com/prometheus/statsd_exporter` (version `v0.22.8`) is now available. Version `v0.22.2` is currently in use.

### PR DESCRIPTION
**Pull request by captncraig**
- github.com/prometheus/statsd_exporter (fixes #2148)
- github.com/prometheus-community/postgres_exporter (fixes #730)
- github.com/prometheus-community/elasticsearch_exporter (fixes #699)(fixes #1970) (adds `aliases` config flag defaulting to true like upstream)
- github.com/prometheus/mysqld_exporter (fixes #1466)
- github.com/prometheus/consul_exporter (fixes #1380)
- github.com/ncabatoff/process-exporter (fixes #1109)
- github.com/prometheus/memcached_exporter (fixes #1813)

### CODE REVIEW

File: pkg/integrations/elasticsearch_exporter/elasticsearch_exporter.go [hunk 1]
```diff
@@ -47,6 +48,8 @@ type Config struct {
 	ExportClusterSettings bool `yaml:"cluster_settings,omitempty"`
 	// Export stats for shards in the cluster (implies indices).
 	ExportShards bool `yaml:"shards,omitempty"`
+	// Include informational aliases metrics
+	IncludeAliases bool `yaml:"aliases,omitempty"`
```
Comment (rfratto) [inline pkg/integrations/elasticsearch_exporter/elasticsearch_exporter.go:52]: Should the docs be updated to include the new field?

**Review by rfratto — COMMENTED** [review]
Nice. Did you check to see if any other config fields need to be added/removed/changed with these updates?

File: go.mod [hunk 2]
```diff
@@ -66,8 +66,8 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/ory/dockertest/v3 v3.8.1
 	github.com/percona/mongodb_exporter v0.31.2
-	github.com/prometheus-community/elasticsearch_exporter v1.2.1
-	github.com/prometheus-community/postgres_exporter v0.10.0
+	github.com/prometheus-community/elasticsearch_exporter v1.5.0
+	github.com/prometheus-community/postgres_exporter v0.11.1
```
Comment (rfratto) [inline go.mod:70]: These are user-facing changes so we should update the CHANGELOG to indicate a version bump. Ditto for statsd_exporter.

File: pkg/integrations/elasticsearch_exporter/elasticsearch_exporter.go (hunk 1 above)
Comment (captncraig) [reply pkg/integrations/elasticsearch_exporter/elasticsearch_exporter.go:52]: Yes they should.

Comment (rfratto) [reply pkg/integrations/elasticsearch_exporter/elasticsearch_exporter.go:52]: I think it goes beyond breaking compilation though, we should see if there's any new options that the exporters expose that we should be propagated to our configs.

Comment (captncraig) [reply pkg/integrations/elasticsearch_exporter/elasticsearch_exporter.go:52]: Good call. There were 2 additional collectors with their flags. And a copy/paste error.

File: docs/sources/configuration/integrations/elasticsearch-exporter-config.md [hunk 3]
```diff
@@ -106,4 +106,18 @@ Full reference of options:
 
   # Skip SSL verification when connecting to Elasticsearch.
   [ ssl_skip_verify: <boolean> ]
+
+  # Include informational aliases metrics.
+  [ aliases: <boolean>]
```
Comment (rfratto) [inline docs/sources/configuration/integrations/elasticsearch-exporter-config.md:111]:
```suggestion
  [ aliases: <boolean> ]
```

File: docs/sources/configuration/integrations/elasticsearch-exporter-config.md [hunk 4]
```diff
@@ -106,4 +106,18 @@ Full reference of options:
...
+
+  # Export stats for Data Streams
+  [ data_stream: <boolean>]
```
Comment (rfratto) [inline docs/sources/configuration/integrations/elasticsearch-exporter-config.md:114]:
```suggestion
  [ data_stream: <boolean> ]
```

File: docs/sources/configuration/integrations/elasticsearch-exporter-config.md [hunk 5]
```diff
@@ -106,4 +106,18 @@ Full reference of options:
...
+
+  # Export stats for SLM (Snapshot Lifecycle Management)
+  [ slm: <boolean>]
```
Comment (rfratto) [inline docs/sources/configuration/integrations/elasticsearch-exporter-config.md:117]:
```suggestion
  [ slm: <boolean> ]
```

File: docs/sources/configuration/integrations/elasticsearch-exporter-config.md [hunk 6]
```diff
@@ -106,4 +106,18 @@ Full reference of options:
 
   # Skip SSL verification when connecting to Elasticsearch.
   [ ssl_skip_verify: <boolean> ]
+
+  # Include informational aliases metrics.
+  [ aliases: <boolean>]
+
+  # Export stats for Data Streams
```
Comment (rfratto) [inline docs/sources/configuration/integrations/elasticsearch-exporter-config.md:113]:
```suggestion
  # Export stats for Data Streams.
```

File: docs/sources/configuration/integrations/elasticsearch-exporter-config.md [hunk 7]
```diff
@@ -106,4 +106,18 @@ Full reference of options:
...
+  [ data_stream: <boolean>]
+
+  # Export stats for SLM (Snapshot Lifecycle Management)
```
Comment (rfratto) [inline docs/sources/configuration/integrations/elasticsearch-exporter-config.md:116]:
```suggestion
  # Export stats for SLM (Snapshot Lifecycle Management).
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -48,6 +48,18 @@ v0.28.0 (2022-09-29)
...
+- Update embedded `github.com/prometheus-community/postgres_exporter` from [`v0.10.0` to `v0.11.1`](https://github.com/prometheus-community/postgres_exporter/releases)
+- Update embedded `github.com/prometheus/memcached_exporter` from [`v0.9.0` to `v0.10.0`](https://github.com/prometheus/memcached_exporter/releases)
+- Update embedded `github.com/prometheus-community/elasticsearch_exporter` from [`v1.2.1` to `v1.5.0`](https://github.com/prometheus-community/elasticsearch_exporter/releases)
+- Update embedded `github.com/prometheus/mysqld_exporter` from [`v0.13.0` to `v0.14.0`](https://github.com/prometheus/mysqld_exporter/releases)
+- Update embedded `github.com/prometheus/consul_exporter` from [`v0.7.2` to `v0.8.0`](https://github.com/prometheus/consul_exporter/releases)
+- Update embedded `github.com/ncabatoff/process-exporter` from [`v0.7.5` to `v0.7.10`](https://github.com/ncabatoff/process-exporter/releases)
+- Update embedded `github.com/prometheus-community/postgres_exporter` from [`v0.10.0` to `v0.11.1`](https://github.com/prometheus-community/postgres_exporter/releases)
```
Comment (rfratto) [inline CHANGELOG.md:60]: List these as `Enhancements` or `Other changes`; we don't have a category called "Updates": https://github.com/grafana/agent/blob/main/docs/developer/contributing.md#updating-the-changelog

**Review by karengermond — APPROVED** [review]
LGTM

**Pull request merged**